src/ORM/Marshaller.php

```diff
@@ -77,6 +78,11 @@ protected function _buildPropertyMap($options)
             $assoc = $this->_table->association($key);
             if ($assoc) {
                 $map[$assoc->property()] = ['association' => $assoc] + $nested + ['associated' => []];
+            } else {
```

Comment:
@Antoniossss Just make a new commit and push to your branch and this PR will be updated.